boards/arm/nrf9160_innblue22/doc/index.rst

```diff
@@ -0,0 +1,106 @@
+.. _nrf9160_innblue22:
+
+nRF9160-INNBLUE22
+#################
+
+Overview
+********
+
+The "innblue v2.2" is a cellular IoT sensor development board, which
+is based on nRF9160 SiP featuring NB-IoT and LTE-M connectivity.
+
+.. figure:: img/nrf9160_innblue22.png
+	 :width: 1024px
+	 :align: center
+	 :alt: nRF9160 innblue22
+
+	 nRF9160 innblue22 (Credit: innblue)
+
+
+Hardware
+********
+
+Following parts are available:
+
+* Accelerometer      > ST LIS2DH12
+* CryptoElement      > Atmel ATECC608a
+* Humidity Sensor    > ST HTS221
+* Qi charger         > TI BQ51013
+* Battery guel gauge > TI BQ27421
+
+
+
+Supported Features
+==================
+
+The nrf9160_innblue22 board configuration supports the following
+hardware (as of nRF9160) features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED1 ( red ) = P0.7
+* LED2 (green) = P0.6
+* LED3 ( blue) = P0.5
+* LED4 ( red ) = P0.4
+
+
+Push buttons and Switches
+-------------------------
+
+* BUTTON1 = P0.31
+
+Security components
+===================
+
+- Implementation Defined Attribution Unit.  The IDAU is implemented
+  with the System Protection Unit and is used to define secure and non-secure
+  memory maps.  By default, all of the memory space  (Flash, SRAM, and
+  peripheral address space) is defined to be secure accessible only.
+- Secure boot.
+
+Building Secure/Non-Secure Zephyr applications
```

Comment:
Please follow the doc/templates/board.tmpl structure throughout, including the "Programming and Debugging" section. The rendered documentation should have exact west commands users can copy/paste to build and flash (if flashing and debugging with west is not possible, then board.cmake has problems). You can see how this is done for nRF9160-DK using `zephyr-app-commands`.